app/scripts/controllers/addConfigVolume.js

```diff
@@ -226,20 +241,26 @@ angular.module('openshiftConsole')
           podTemplate.spec.volumes = podTemplate.spec.volumes || [];
           podTemplate.spec.volumes.push(newVolume);
 
-          // Clear any previous alerts.
-          $scope.alerts = {};
-
           $scope.disableInputs = true;
+          hideErrorNotifications();
+
+          var humanizeKind = $filter('humanizeKind');
+          var sourceKind = humanizeKind(source.kind);
+          var targetKind = humanizeKind($routeParams.kind);
+
           DataService.update(resourceGroupVersion, resource.metadata.name, $scope.targetObject, context).then(
             function() {
-              _.set($scope, 'confirm.doneEditing', true);
-              $window.history.back();
+              NotificationsService.addNotification({
+                type: "success",
+                message: "Succesfully added " + sourceKind + " " + source.metadata.name +
+                         " to " + targetKind + " " + $routeParams.name + ".",
+                details: "Files mounted at path " + mountPath
```

Comment:
should probably be a period at the end of the details sentence, and maybe say "The files will be mounted at the path \<whatever\>.